Templates/GitlabCIPipeline/.gitlab-ci.yml

```diff
@@ -659,9 +661,9 @@ Create production version:
                 curl --request POST --header "PRIVATE-TOKEN: ${AutomationToken}" "${CI_API_V4_URL}/projects/${CI_PROJECT_ID}/repository/branches?branch=release%2Frel-${newVersionTag}&ref=rel-${newVersionTag}"
                 RC=$?
                 if [ ${RC} -eq 0 ]; then
-                    echo [INFO] Created release maintenance: ${newVersionTag}.
+                    echo [INFO] Created release maintenance branch: ${newVersionTag}.
```

Comment:
The branch name will be

`release/rel-${newVersionTag}`  and not `${newVersionTag}`.

@M-DLB Can you please update the info message?